src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FILE_NAME = 0;
```

Comment:
bad naming, it's not name. it's not even file. dont invent your own names where there are existing ones
```suggestion
    private static final int OPERATION_TYPE_INDEX = 0;
```